ports/libsimpleservo/capi/src/vslogger.rs

```diff
@@ -2,17 +2,61 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
 
+use crate::LOG_MODULE_FILTERS;
 use log::{self, Level, Metadata, Record};
+use std::ffi::CStr;
+use std::os::raw::c_char;
 
 extern "C" {
     fn OutputDebugStringA(s: *const u8);
 }
 
+pub struct VsloggerModuleFilter {
```

Comment:
Let's remove this struct, since it just wraps a vector.

Reply:
Structure removed.  Variable now defined as:
static mut vslogger_filter: Vec<String> = vec![];

Reply:
We should still be using the lazy_static so we can use the Arc<Mutex> wrapper.

Reply:
After struggling I found I needed it. :)  Noticed I had to add "pub" too.   This is what I have now:

```
lazy_static! {
    pub static ref vslogger_filter: Vec<String> = vec![];
}
```

Reply:
Implemented.